### PR TITLE
pass on LOGLEVEL to tool invocation

### PR DESCRIPTION
**Pull request by kba**
So one can do

```
ocrd-make -f ...mk LOGLEVEL=DEBUG
```

to see debug logging.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -275,7 +277,7 @@ ifneq ($(wildcard $(CURDIR)/mets.xml),)
 space = $() $()
 comma = ,
 define toolrecipe =
-$(TOOL) -I $(subst $(space),$(comma),$^) -O $@ -p $@.json 2>&1 | tee $@.log && \
+$(TOOL) -l $(LOGLEVEL) -I $(subst $(space),$(comma),$^) -O $@ -p $@.json 2>&1 | tee $@.log && \
```
Comment (bertsky) [inline Makefile]: The command-line `-l` is an _override_ to all logging defaults and config settings, so I cannot use this verbatim. 

```suggestion
$(TOOL) $(and $(LOGLEVEL),-l $(LOGLEVEL)) -I $(subst $(space),$(comma),$^) -O $@ -p $@.json 2>&1 | tee $@.log && \
```

Okay?

File: Makefile [hunk 2]
```diff
@@ -30,6 +30,8 @@ VIRTUAL_ENV ?= $(CURDIR)/local
 BINDIR = $(abspath $(VIRTUAL_ENV))/bin
 # copy the makefiles here:
 SHAREDIR = $(abspath $(VIRTUAL_ENV))/share/workflow-configuration
+# log level
+LOGLEVEL ?= INFO
```
Comment (bertsky) [inline Makefile]:
```suggestion
```

(see other comment)

**Review by bertsky — CHANGES_REQUESTED** [review]
Good idea. (Maybe I should also support a `-g` opt-in for the desparate.)

File: Makefile (hunk 1 above)
Comment (kba) [reply Makefile]: Sure

Comment (kba) [reply Makefile]: `$(and $(LOGLEVEL),-l $(LOGLEVEL)` is a really elegant idiom for Makefiles btw.

**Review by bertsky — APPROVED** [review]

**Pull request merged**